lib/simplifyExpression/basicsSearch/index.js

```diff
@@ -25,6 +26,8 @@ const SIMPLIFICATION_FUNCTIONS = [
   reduceExponentByZero,
   // Check for x^1 which should be reduced to x
   removeExponentByOne,
+  // Check for 1^x which should be reduced to 1
```

Comment:
maybe add something to the end of this comment ..."if x can be simplified to a constant" to be extra clear about this case